src/OrchardCore.Modules/OrchardCore.UrlRewriting/Drivers/RewriteRuleDisplayDriver.cs

```diff
@@ -0,0 +1,111 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Localization;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.UrlRewriting.Models;
+using OrchardCore.UrlRewriting.ViewModels;
+using OrchardCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using OrchardCore.UrlRewriting.Services;
+using OrchardCore.UrlRewriting.Rules;
+using OrchardCore.UrlRewriting.Helpers;
+
+namespace OrchardCore.UrlRewriting.Drivers;
+
+internal sealed class RewriteRuleDisplayDriver : DisplayDriver<RewriteRule>
```

Comment:
our drivers are public incase someone want to remove it from the IoC.

```suggestion
public sealed class RewriteRuleDisplayDriver : DisplayDriver<RewriteRule>
```